src/posts/tracker/changelog.ts

```diff
@@ -15,6 +15,72 @@ interface Change {
 // NOTE: Do not include '/edit' at the end of the URL, because we append different paths like /copy or /template/preview depending on the use case. Do not include a slash at the end of the URL.
 
 const changes: Change[] = [
+  {
+    date: new Date(2021, 7, 22),
+    versionNum: '2.4',
+    title: 'Increase podmate budgets if everyone is vaccinated',
+    spreadsheetURL:
+      'https://docs.google.com/spreadsheets/d/1Es4ZzLlNiBSxG5jJsGPUYewrCw2NqB5kDXlNfmdSdD0',
+    whatsNew: `
+* New: Low the budget reductions from other podmates if everyone in the pod is vaccinated since transmission is lower. Follows the [formula outlined here](/paper/13-q-and-a#2-if-a-vaccinated-individual-contracts-covid-how-much-less-or-more-likely-is-this-to-result-in-negative-consequences-increased-budget). ([#1013](https://github.com/microCOVID/microCOVID/issues/1013))
```

Comment:
"Low the budget reductions from other podmates"

I find the double negative to be confusing, would vaguely prefer "Increase individual budgets"

"transmission is lower" -> "transmission rate is lower" or "household transmission is less likely"

Reply:
done